src/fenwick.jl

```diff
@@ -0,0 +1,37 @@
+mutable struct FenwickTree{T}
+    BIT::Vector{T}
+    n::Integer
+end
+
+FenwickTree{T}() where T = FenwickTree{T}(0)
```

Comment:
Should there be a constructor that takes a iterator if items and fills the tree?

Reply:
Umm, no. I thought about it, but the tree is actually is the representation of an array, you can't just have a random array be a BIT and then expect some proper functions out of it. That being said, there is always a scope for its inclusion. Maybe, I'll include it to make it look more generic.